src/styles/blocks/card.scss

```diff
@@ -0,0 +1,111 @@
+.main {
+  display: flex;
+  box-sizing: border-box;
+  flex-wrap: wrap;
+  padding: 50px 40px;
+  margin: 0 auto;
+  width: 1024px;
+}
+
+.main__card {
+  margin-right: 48px;
+  margin-bottom: 46px;
+
+  &:nth-child(4n) {
+    margin-right: 0;
+  }
+
+  &:nth-last-child(-n + 4) {
+    margin-bottom: 0;
+  }
+}
+
+.card {
```

Comment:
You can't write styles for main inside card.scss
<img width="975" alt="image" src="https://user-images.githubusercontent.com/99403198/222709149-366053cc-b5d0-4fe5-925a-b70b07fe6fe3.png">